starcheck/calc_ccd_temps.py

```diff
@@ -152,33 +150,32 @@ def get_ccd_temps(oflsdir, outdir='out',
     except TypeError:
         sc_obsids = json.load(json_obsids)
 
-    tnow = DateTime().secs
-
     # Get tstart, tstop, commands from backstop file in opt.oflsdir
     bs_cmds = get_bs_cmds(oflsdir)
-    tstart = bs_cmds[0]['time']
-    tstop = bs_cmds[-1]['time']
+    tstart = DateTime(bs_cmds[0]['date']).secs
+    tstop = DateTime(bs_cmds[-1]['date']).secs
     proc['datestart'] = DateTime(tstart).date
     proc['datestop'] = DateTime(tstop).date
 
-
-    # Get temperature telemetry for 30 days prior to
-    # min(tstart, NOW, run_start_time) where run_start_time is basically
-    # a mock NOW for regression testing.
-    tlm = get_telem_values(min(tstart, tnow, run_start_time.secs),
-                           ['aacccdpt', 'pitch'],
-                           days=30)
+    # Get temperature telemetry for 1 days prior to
+    # min(last available telem, backstop tstart, NOW, run_start_time)
+    # where run_start_time is basically a mock NOW for regression testing.
+    msid_range = fetch.get_time_range('aacccdpt', format='secs')
```

Comment:
@taldcroft I think this is a real change relative to your kadi-states branch.  Do you want me to try to make this more readable?

Reply:
I think you can cut NOW.  I don't see how telemetry can extend past NOW.  Otherwise OK.  I do generally prefer plural names for lists e.g. `msid_times`, or use `end_time = fetch.get_time_range(..)[1]`.

Reply:
Good point about telemetry not extending into the future!